bridge/slack/slack.go

```diff
@@ -471,13 +472,22 @@ func (b *Bslack) uploadFile(msg *config.Message, channelID string) {
 		})
 		if err != nil {
 			b.Log.Errorf("uploadfile %#v", err)
-			return
+			return "", err
 		}
 		if res.ID != "" {
 			b.Log.Debugf("Adding file ID %s to cache with timestamp %s", res.ID, ts.String())
 			b.cache.Add("file"+res.ID, ts)
+
+			//search for message id by uploaded file in private/public channels, get thread timestamp from uploaded file
```

Comment:
```suggestion
			// search for message id by uploaded file in private/public channels, get thread timestamp from uploaded file
```

(to fix the gofumpt lint)